yarn.lock

```diff
@@ -17915,6 +17915,11 @@ ses@^0.14.3:
   resolved "https://registry.yarnpkg.com/ses/-/ses-0.14.3.tgz#fadc5a3baa0878adfcbdc8fa383218a821974270"
```

Comment:
Why wasn't `ses-0.14.3` removed from the `yarn.lock`?  Does something depend on it explicitly?

Reply:
> Why wasn't `ses-0.14.3` removed from the `yarn.lock`? Does something depend on it explicitly?

Ah, looks like `packages/governance/package.json` references `ses` when it shouldn't.

Reply:
I’ve synced up governance. This is probably something that slipped in with a rebase.